### PR TITLE
Add Bowens

### PR DESCRIPTION
**Pull request by dev-minjin-chou**

### CODE REVIEW

File: entries/b/bowens.com.au.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "Bowens":{
+      "domain":"bowens.com.au",
+      "recovery": "https://www.bowens.com.au/my-account/login?redirect=%2F",
```
Comment (Carlgo11) [inline entries/b/bowens.com.au.json]: The recovery element is intended to provide users documentation on how accounts can be recovered. Not an actual link to the recovery page.
```suggestion
```

File: img/b/bowens.com.au.svg [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    </g>
+</svg>
```
Comment (Carlgo11) [inline img/b/bowens.com.au.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" style="fill:#fff200;background:#154c97"><path d="M14.1 9.55c-.4.3-1 .5-2 .5H9.5v-3.7h2.7c.9 0 1.6.2 1.9.5.4.3.5.8.5 1.3.1.6-.1 1-.5 1.4m-4.6 4.8h3.1c1.1 0 1.8.2 2.3.5.4.4.6.8.6 1.4 0 .6-.2 1.1-.6 1.5-.4.4-1.2.6-2.2.6H9.6Zm8.6-2.6c.8-.3 1.5-.8 2-1.3.8-.9 1.2-2.1 1.2-3.3 0-1.5-.6-2.9-1.7-3.9s-2.7-1.6-4.7-1.6H2.7v21.7H14c.5 0 1.5-.1 2.9-.3 1.1-.1 1.9-.4 2.5-.7.9-.5 1.6-1.3 2.1-2.2.5-.9.8-1.9.8-3.1 0-1.4-.4-2.6-1.1-3.5-.7-.8-1.7-1.4-3.1-1.8"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/b/bowens.com.au.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      "tfa": [
+        "sms",
+        "email"
```
Comment (Carlgo11) [inline entries/b/bowens.com.au.json:6]: I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

**Comment by dev-minjin-chou** [comment]
As 2FA are enabled on default during account sign up process. There is no way to disable 2FA by user, and no documentation regarding it. Screenshots below outline the login process of verified user.

## Process:
- Logging in: 
image

- 2FA: 
image

- After submitting authentication code: 
image

File: entries/b/bowens.com.au.json (hunk 3 above)
Comment (dev-minjin-chou) [reply entries/b/bowens.com.au.json:6]: Could not find relevant documentation, screenshots are attached below. Cheers 🚀

**Review by Carlgo11 — APPROVED** [review]
Looks good. Thanks for the pull request!

**Pull request merged**